From 7af62198070d88dc58aca18ff4254c6ac9367d17 Mon Sep 17 00:00:00 2001
From: Imre Kaloz <kaloz@openwrt.org>
Date: Thu, 16 Mar 2023 10:24:35 +0530
Subject: [PATCH 017/281] OpenWrt:
 261-enable_wilink_platform_without_drivers.patch

hack: net: wireless: make the wl12xx glue code available with
 compat-wireless, too

Signed-off-by: Imre Kaloz <kaloz@openwrt.org>

Change-Id: Ic2660da21b8e4d0167ab17250512d1cdbf3fa8c7
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
---
 drivers/net/wireless/ti/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ti/Kconfig b/drivers/net/wireless/ti/Kconfig
index 7c0b17a76fe2..9f0edc5aca27 100644
--- a/drivers/net/wireless/ti/Kconfig
+++ b/drivers/net/wireless/ti/Kconfig
@@ -20,7 +20,7 @@ source "drivers/net/wireless/ti/wlcore/Kconfig"
 
 config WILINK_PLATFORM_DATA
 	bool "TI WiLink platform data"
-	depends on WLCORE_SDIO || WL1251_SDIO
+	depends on WLCORE_SDIO || WL1251_SDIO || ARCH_OMAP2PLUS
 	default y
 	help
 	Small platform data bit needed to pass data to the sdio modules.
-- 
2.17.1

